backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -64,7 +66,12 @@ private CassandraCluster(CassandraModule module, Host host) throws RuntimeExcept
             .keyspace(KEYSPACE)
             .replicationFactor(1)
             .disableDurableWrites();
-        this.nonPrivilegedCluster = ClusterFactory.create(clusterConfiguration, keyspaceConfiguration);
+
+        DriverConfigLoader driverConfigLoader = DriverConfigLoader.programmaticBuilder()
+            .withString(DefaultDriverOption.SESSION_LEAK_THRESHOLD, "16")
```

Comment:
Why do we have that many session actives and does it affect perf/stability? I find a bit easy to just silence it?

Reply:
> Why do we have that many session actives and does it affect perf/stability

I find a test creates a new session for each test and not close to it, I think we have more similarities somewhere 

When I read a log from `test-run.log` of the fail test case, I see a suspect warning
```

21:08:37.630 [WARN ] c.d.o.d.i.c.session.DefaultSession - You have too many session instances: 10 active, expected less than 4 (see 'advanced.session-leak.threshold' in the configuration)
21:08:37.981 [WARN ] c.d.o.d.i.c.session.DefaultSession - You have too many session instances: 11 active, expected less than 4 (see 'advanced.session-leak.threshold' in the configuration)
```

Reply:
CI still fails, It looks not resolve the unstable -> Reverted